homeworks/src/ltretykhlib/module3/Triangle.java

```diff
@@ -0,0 +1,98 @@
+package ltretykhlib.module3;
+
+import static java.lang.Math.*;
+
+/**
+ * Created by Tretykhlib on 23.04.2017.
+ */
+public class Triangle {
+    private double Ax;
+    private double Ay;
+    private double Bx;
+    private double By;
+    private double Cx;
+    private double Cy;
+
+    public Triangle(double axTop, double ayTop, double bxTop, double byTop, double cxTop, double cyTop) {
+        Ax = axTop;
+        Ay = ayTop;
+        Bx = bxTop;
+        By = byTop;
+        Cx = cxTop;
+        Cy = cyTop;
+    }
+
+    public void setAx(double ax) {
+        Ax = ax;
+    }
+
+    public void setAy(double ay) {
+        Ay = ay;
+    }
+
+    public void setBx(double bx) {
+        Bx = bx;
+    }
+
+    public void setBy(double by) {
+        By = by;
+    }
+
+    public void setCx(double cx) {
+        Cx = cx;
+    }
+
+    public void setCy(double cy) {
+        Cy = cy;
+    }
```

Comment:
In my opinion, a triangle should be an immutable object ( can't be changed).
Considering said above, no setters needed, the only way to construct an instance is using the constructor